storage/queue.go

```diff
@@ -0,0 +1,223 @@
+package storage
+
+import (
+	"encoding/xml"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+)
+
+// QueueServiceClient contains operations for Microsoft Azure Queue Storage
+// Service.
+type QueueServiceClient struct {
+	client Client
+}
+
+func pathForQueue(name string) string          { return fmt.Sprintf("/%s", name) }
+func pathForQueueMessages(name string) string  { return fmt.Sprintf("/%s/messages", name) }
+func pathForMessage(queue, name string) string { return fmt.Sprintf("/%s/messages/%s", queue, name) }
```

Comment:
nit: I would rename `name` parameter to `queue` in the top 2 of these for consistency